framework/src/modules/chain/dpos/dpos.js

```diff
@@ -0,0 +1,15 @@
+const Delegates = require('./delegates');
+
+const __methods = {
```

Comment:
prefer not to use weakmap. in Node 12 there will be private method, but we shouldnt use it now